### PR TITLE
Dprice azure marketplace topic

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-azure-marketplace-topic-neondatabase.vercel.app/docs/introduction/billing-azure-marketplace

### CODE REVIEW

File: content/docs/introduction/billing-azure-marketplace.md [hunk 1]
```diff
@@ -39,19 +39,15 @@ After subscribing, you will be directed to the [Create a Neon Resource](#create-
...
 
-TBD
+## Changing your pricing plan
 
-## Changing your billing plan
-
-You can change your Neon Plan at any time. To do so ...
-
-TBD
+To change your Neon pricing plan, you can cancel your current subscription and create a new one. However, this approach does not preserve your existing Neon projects. If you want to keep your projects while changing plans, please open a [support ticket](https://console.neon.tech/app/projects?modal=support). The Neon Support team will help you update your plan and migrate your projects to the new plan.
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/introduction/billing-azure-marketplace.md]: yes! Thank you

File: content/docs/introduction/billing-azure-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   | **Resource Name**          | Enter the name for the Neon organization you want to create.                                                                                                                                                                                                                                                       |
+   | **Region**                 | Select a region to deploy your resource. Neon Serverless Postgres will let you choose a region while creating a project.                                                                                                                                                                                           |
+   | **Neon Organization name** | Corresponds to the name of your company, usually.                                                                                                                                                                                                                                                                  |
```
Comment (mocasii) [inline content/docs/introduction/billing-azure-marketplace.md:36]: Is this what we recommend? Neon "Organization" isn't currently in the Neon Docs Glossary page.

File: content/docs/introduction/billing-azure-marketplace.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   | **Subscription**           | From the drop-down, select your Azure subscription where you have Owner or Contributor access.                                                                                                                                                                                                                     |
+   | **Resource group**         | Specify whether you want to create a new resource group or use an existing one. A resource group is a container that holds related resources for an Azure solution. For more information, see [Azure Resource Group overview](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/overview). |
+   | **Resource Name**          | Enter the name for the Neon organization you want to create.                                                                                                                                                                                                                                                       |
```
Comment (mocasii) [inline content/docs/introduction/billing-azure-marketplace.md:34]: Can we make this clearer that it's the Azure resource name? This field will only be used on the Azure side.

File: content/docs/introduction/billing-azure-marketplace.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+   | **Region**                 | Select a region to deploy your resource. Neon Serverless Postgres will let you choose a region while creating a project.                                                                                                                                                                                           |
+   | **Neon Organization name** | Corresponds to the name of your company, usually.                                                                                                                                                                                                                                                                  |
+   | **Pricing Plan**           | Choose a plan of your choice. For information about Neon's plans, please refer to the [Neon Pricing](https://neon.tech/home) page. Neon offers a Free Plan and paid plans (**Scale**, and **Business**) that you can choose from.                                                                                  |
```
Comment (mocasii) [inline content/docs/introduction/billing-azure-marketplace.md:37]: > Choose a plan of your choice
Change to "Select a plan"?

File: content/docs/introduction/billing-azure-marketplace.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+    <Admonition type="note">
+    The Neon **Launch Plan** is currently not supported.
```
Comment (mocasii) [inline content/docs/introduction/billing-azure-marketplace.md:40]: "Currently not available in the Azure Marketplace".

File: content/docs/introduction/billing-azure-marketplace.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/introduction/billing-azure-marketplace.md:36]: I've expanded on that with a link to a glossary term.

File: content/docs/introduction/billing-azure-marketplace.md (hunk 4 above)
Comment (danieltprice) [reply content/docs/introduction/billing-azure-marketplace.md:37]: fixed.

File: content/docs/introduction/billing-azure-marketplace.md (hunk 5 above)
Comment (danieltprice) [reply content/docs/introduction/billing-azure-marketplace.md:40]: fixed.

**Pull request merged**